package.json

```
@@ -38,7 +38,8 @@
     "visual-testing-app:build": "yarn workspace visual-testing-app build",
     "visual-testing-app:preview": "yarn workspace visual-testing-app preview",
     "labels:sync": "github-labels sync",
-    "typecheck": "tsc --noEmit --skipLibCheck"
+    "typecheck": "tsc --noEmit --skipLibCheck",
+    "omlet-scan": "yarn omlet analyze --hook-script ./scripts/omlet-hook-script.js"
```

```suggestion
    "omlet:scan": "yarn omlet analyze --hook-script ./scripts/omlet-hook-script.js"
```
separating the noun from the verb with a colon seems to be the norm for these scripts, my care amount is relatively low but consistency for things like script names really makes a difference as we add more scripts.

---

hahaha, yeah, why not